### ISSUE TITLE
Better documentation for integrating into a JAMStack site, as well as integrating with popular static hosting providers such as Netlify

### ISSUE THREAD

**Issue by johncmunson** [posativ/isso#508]
The documentation seems to be quite sparse, especially when compared with the experience [this fella](https://stiobhart.net/2017-02-24-isso-comments/) went through just to get a working solution.

This limited amount of documentation may be just fine for people with plenty of experience with web servers like nginx, but for a frontend developer like myself the existing documentation just doesn't cut it. I imagine this is true for most people just getting into JAMStack/static websites and happen to come across this project.

This is unfortunate, since this project is primarily meant to be used with static websites. I think we would all very much benefit from some more thorough documentation and some guides showing how to integrate with some of the popular static hosting providers, such as Netlify.

**Comment by jelmer** [comment]
Agreed. Help to improve the documentation and cover examples of how to use issue with various reverse proxies or static blog generators would be very welcome.
